docs/graphql/manual/guides/deployment/render.rst

```diff
@@ -0,0 +1,115 @@
+Hasura GraphQL Engine on Render
+====================================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide talks about how to deploy the Hasura GraphQL engine on `Render
+<https://render.com>`__.
+
+
+Pre-requisites
+--------------
+
+- GitHub (or alternative) account
+- Render account associated with a Github account and with billing enabled
+
+
+Create a repository
+-------------------
+
+Create a new GitHub repository
+
+Within the new repository, create a new file named ``Dockerfile`` that 
+references the latest Hasura GraphQL Engine docker image by including the 
+following single line in the file:
+
+.. code-block:: bash
+
+   FROM hasura/graphql-engine:latest
+
+
+Using Render Infrastrcuture as Code (Fastest)
```

Comment:
Not sure what is meant by "using Render infrastructure as code"

Reply:
This is a capabiltiy of Render. I'm not sure if we should refer to it as a product name "Infrastructure as Code" (capitalized proper noun) or just a feature description (lowercase)

Reply:
I think we can change this to Using `render.yaml` (fastest)